cluster-up/cluster/kind/common.sh

```diff
@@ -155,7 +184,7 @@ function setup_kind() {
     echo "ipv6 cni: $IPV6_CNI"
     if [ -z ${IPV6_CNI+x} ]; then
         echo "no ipv6, safe to install calico"
-        _kubectl apply -f $KIND_MANIFESTS_DIR/kube-calico.yaml
+        _kubectl apply -f "$(_patch_calico_manifest "$KIND_MANIFESTS_DIR/kube-calico.yaml.in" "$KIND_MANIFESTS_DIR/kube-calico.diff.in" "/tmp")"
```

Comment:
I think this is wrong, if you are going to use stdin you have to use `kubectl apply -f -` after a pip, also remove all those arguments to `_patch_calico_manifest` we don't need a generic function there
```suggestion
        _patch_calico_manifest | _kubectl apply -f -
```

Reply:
Done